data_store/controllers/notify.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+from notifications_python_client.errors import HTTPError
+from notifications_python_client.notifications import NotificationsAPIClient
+from werkzeug.datastructures import FileStorage
+
+from config import Config
+from data_store.controllers.load_functions import get_or_generate_submission_id
+from data_store.db.queries import get_programme_by_id_and_round
+from submit.main.fund import FundConfig
+from submit.utils import get_friendly_fund_type
+
+
+def send_email(
```

Comment:
Deleted the part that accepted attachments. Maybe controversial. We don't use it anymore. Can be added again if needed.

Reply:
Makes sense to me.

The type hints of `notify_key: str | None` feels wrong - it's a required value, so should just be `str` now

Reply:
`mypy` shamed me for it.
 
Because the NOTIFY_KEY envs can be `None` as we currently fetch them from config. I can change that..

Reply:
Ah yeah... this sort of ties into https://dluhcdigital.atlassian.net/browse/FPASF-163 I guess. It being optional config is annoying...